### ISSUE TITLE
Starting saved machine not working in Virtualbox

### ISSUE THREAD

**Issue by erichonkanen** [docker/machine#1523]
I have a machine that is in the saved state and I cannot restart it. When I do `docker-machine -D start mymachine`, I get error I pasted at bottom of this:

https://github.com/boot2docker/boot2docker-cli/issues/292

**Comment by chouclee** [comment]
For me, it never works. I always re-create a new machine if it is in `saved` state. 
Mac, machine version 0.3.0

**Comment by erichonkanen** [comment]
@chouclee I was able to use the virtualbox dashboard to power off the machine and then use docker-machine to start it

**Comment by chouclee** [comment]
@erichonkanen I never tried to start it via GUI. I can try your solution next time. Thanks.
 "it never works" I mean by command line `docker-machine start --`.

**Comment by saurabh-g** [comment]
I cannot start the machine after it is in the saved state. This is the error I get:

```
STDERR:
executing: /usr/bin/VBoxManage modifyvm dev --nic2 hostonly --nictype2 82540EM --hostonlyadapter2 vboxnet2 --cableconnected2 on
STDOUT:
STDERR: VBoxManage: error: The machine is not mutable (state is Saved)
VBoxManage: error: Details: code VBOX_E_INVALID_VM_STATE (0x80bb0002), component SessionMachine, interface IMachine, callee nsISupports
VBoxManage: error: Context: "COMSETTER(Enabled)(TRUE)" at line 1448 of file VBoxManageModifyVM.cpp
VBoxManage: error: The machine is not mutable (state is Saved)
VBoxManage: error: Details: code VBOX_E_INVALID_VM_STATE (0x80bb0002), component SessionMachine, interface IMachine, callee nsISupports
VBoxManage: error: Context: "COMSETTER(AttachmentType)(NetworkAttachmentType_HostOnly)" at line 1449 of file VBoxManageModifyVM.cpp
```

**Comment by erichonkanen** [comment]
@saurabh-g so far the workaround is use the VB gui to power off the machine then start it again

**Comment by saurabh-g** [comment]
@erichonkanen  Yeah, we have a start up script that sets up docker machine as a dev environment. I use `vboxmanage controlvm dev poweroff` as a part of the script. That does the trick too.

**Comment by sergiobuj** [comment]
There seems to be a fix for this already in master with commit https://github.com/docker/machine/commit/9053ee24c79b38e757652484197678ee76117de7

**Comment by dgageot** [comment]
Closed by 9053ee24c79b38e757652484197678ee76117de7
